code/game/machinery/computer/message.dm

```diff
@@ -15,7 +15,6 @@
 	//Server linked to.
 	var/obj/machinery/message_server/linkedServer = null
 	//Sparks effect - For emag
-	var/datum/effect_system/spark_spread/spark_system = new /datum/effect_system/spark_spread
```

Comment:
Having a dedicated holder for something that only occurs on emagging? What? No.

Reply:
lol